app/views/devise/mailer/email_changed.html.haml

```diff
@@ -0,0 +1,8 @@
+%p
+  Hello #{@email}!
```

Comment:
...the default tone of their emails feels off.

Reply:
added a todo, will get to this as part of the PR and ask you for feedback!